tests/ui/unwrap.rs

```diff
@@ -10,7 +10,49 @@ fn unwrap_result() {
     let _ = res.unwrap();
 }
 
+fn unwrap_get() {
+    let v = vec![1, 2, 3];
+    let _ = v.get(0).unwrap();
+}
+
+fn unwrap_get_mut() {
+    let mut v = vec![1, 2, 3];
+    let _ = v.get_mut(0).unwrap();
+}
+
+#[warn(clippy::get_unwrap)]
+fn unwrap_get2() {
+    let v = vec![1, 2, 3];
+    let _ = v.get(0).unwrap();
+}
+
+#[warn(clippy::get_unwrap)]
+fn unwrap_get_mut2() {
+    let mut v = vec![1, 2, 3];
+    let _ = v.get_mut(0).unwrap();
+}
+
+#[warn(clippy::get_unwrap)]
+#[allow(clippy::unwrap_used)]
+fn unwrap_get3() {
+    let v = vec![1, 2, 3];
+    let _ = v.get(0).unwrap();
+}
+
+#[warn(clippy::get_unwrap)]
+#[allow(clippy::unwrap_used)]
+fn unwrap_get_mut3() {
+    let mut v = vec![1, 2, 3];
+    let _ = v.get_mut(0).unwrap();
+}
```

Comment:
You don't have to tet `get_unwrap` in this file. It's tested in `tests/ui/get_unwrap.rs`.